src/components/BulkManagementHistoryView/index.jsx

```diff
@@ -12,7 +12,7 @@ import HistoryTable from './HistoryTable';
  */
 export const BulkManagementHistoryView = () => (
   <div className="bulk-management-history-view">
-    <h4><FormattedMessage {...messages.heading} /></h4>
+    <h4 className="font-weight-bold mt-0"><FormattedMessage {...messages.heading} /></h4>
```

Comment:
[curious] Why was it necessary to add `font-weight-bold` to a heading element? Our typography styles for heading in Paragon default to `font-weight: 700`, so including this CSS utility class should be unnecessary.